docs/learn/avalanche/avalanche-consensus.md

```diff
@@ -1,45 +1,81 @@
 ---
-description: Consensus is the task of getting a group of computers to come to an agreement on a decision. Avalanche is a new consensus protocol that is scalable, robust, and decentralized.
+description: Avalanche is a new consensus protocol that is scalable, robust, and decentralized.
```

Comment:
While we're changing this: should we still describe it as "new" given that it came out in 2019?